plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/DeltaLakeMetadata.java

```diff
@@ -1011,9 +1164,28 @@ private static Map<String, String> deltaTableProperties(ConnectorSession session
             // Mimicking the behavior of the Hive connector which sets both `Table#setTableType` and the "EXTERNAL" table property
             properties.put("EXTERNAL", "TRUE");
         }
+        if (cacheTableMetadata) {
+            tableComment.ifPresent(comment -> properties.put(TABLE_COMMENT, comment));
+            properties.put(TRINO_LAST_TRANSACTION_VERSION, Long.toString(version));
+            properties.put(TRINO_METADATA_SCHEMA_STRING, schemaString);
+        }
```

Comment:
Extract to utility method.

I see the very same code used at the lines 1179-1183